src/app/static/src/app/store/translations/locales.ts

```diff
@@ -517,6 +541,42 @@ const en: Translations = {
     population: "Population",
     preparing: "Preparing files...",
     previousPage: "Previous page",
+    privacyPreambleHeading: "Privacy notice for naomi.unaids.org",
+    privacyPreambleBody: "In accordance with data protection legislation and the privacy electronic communications regulations, the privacy notice will:",
+    privacyPreambleList: "<li>inform you as to how we look after your personal data when you visit our websites;</li>\n" +
+        "                 <li>tell you about your privacy rights;</li>\n" +
+        "                 <li>tell you how the law protects you;</li>\n",
+    privacyPreambleBody2: "This policy is based on the <a href=\"https://www.imperial.ac.uk/about-the-site/privacy/\" target='_blank' rel='noopener noreferrer'>Imperial College privacy policy</a>.",
+    privacyCollectHeading: "What do we collect?",
+    privacyCollectBody: "If you use this application we will ask you to provide:",
+    privacyCollectList: "<li>Your email address</li>" +
+        "                <li>Input data required for the model to produce estimates including Spectrum PJNZ, country geojson, population data, survey data, ART data and ANC data</li>" +
+        "                <li>Parameters for running a model fit</li>",
+    privacyThirdPartyHeading: "Information we collect from third parties",
+    privacyThirdPartyList: "<li>If you login to Naomi using the HIV Tools single sign-on service you direct the service to send us profile information. Your email address will be stored on the Naomi servers.</li>" +
```

Comment:
Perhaps single sign-on should be capitalised as it is on the login page.. "Log in with your HIV Tools Single Sign-On account?